src/plonemeeting/portal/core/filters/replace_masked_gdpr.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+from plone import api
+from plone.api.portal import get_navigation_root
+from plone.api.portal import get_registry_record
+from plone.outputfilters.interfaces import IFilter
+from plonemeeting.portal.core.config import DELIB_ANONYMIZED_TEXT
+from plonemeeting.portal.core.config import RGPD_MASKED_TEXT
+from zope.interface import implementer
+
+import re
+
+
+@implementer(IFilter)
+class ReplaceMaskedGDPR(object):
+    order = 1000
+
+    def __init__(self, context, request):
+        self.context = context
+        self.institution = get_navigation_root(self.context)
+        self.request = request
+
+    def is_enabled(self):
+        return True
+
+    def __call__(self, data):
+        to_replace = get_registry_record("plonemeeting.portal.core.delib_masked_gdpr", default=DELIB_ANONYMIZED_TEXT)
+        if not to_replace:  # get_registry_record may return None if record exists but empty
+            to_replace = DELIB_ANONYMIZED_TEXT
+
+        if hasattr(self.institution, "url_rgpd") and self.institution.url_rgpd:
```

Comment:
@odelaere j'enlèverais les "hasattr(self.institution, "url_rgpd"), l'institution doit l'avoir d'office puisque c'est dans son schema, donc "if self.institution.url_grpd is not None" sera plus clair

Reply:
Il me semblait qu'un migration ou une édition était nécessaire pour init les attr mais je vois que c'est pas le cas